jest.config.js

```diff
@@ -9,6 +9,7 @@ export default {
 				'^@/(.*)$': '<rootDir>/src/$1',
 				'^(\\.{1,2}/.*)\\.js$': '$1',
 			},
+			setupFilesAfterEnv: ['./test/jest.setup-file.ts'],
```

Comment:
note to reviewer: I use AfterEnv, because miniflare exposes `crypto`, and the setup is modifying `crypto.subtle.sign`

Reply:
I recall there was an easier way using Jest for mocking WebCrypto API.
See: https://github.com/cloudflare/voprf-ts/blob/main/test/server.test.ts#L34-L53

Reply:
it can be done with jest spyOn, but given this is a platform behaviour, and not an internal method we're trying to listen on, I'm inclined to have it in the setup files and replace the global.
Another option is to update miniflare environment package, but that's deprecated, without a clear new replacement for jest